lib/resources/styles.css

```diff
@@ -110,12 +115,18 @@ nav.navbar {
   border: 0;
 }
 
-@media (max-width: 767px) {
+@media (max-width: 768px) {
   .hidden-xs {
     display: none!important;
```

Comment:
not part of this PR, but this needs a space before `!important`

Reply:
done